system/nxplayer/nxplayer.c

```diff
@@ -121,6 +131,27 @@ static const struct nxplayer_ext_fmt_s g_known_ext[] =
 
 static const int g_known_ext_count = sizeof(g_known_ext) /
                     sizeof(struct nxplayer_ext_fmt_s);
+
+static const struct nxplayer_dec_ops_s g_dec_mp3 =
+{
+  AUDIO_FMT_MP3,
+  nxplayer_parse_mp3,
+  nxplayer_fill_mp3
+};
+
+static const struct nxplayer_dec_ops_s g_dec_pcm =
+{
+  AUDIO_FMT_PCM,
+  NULL,
+  nxplayer_fill_pcm
+};
+
+static const struct nxplayer_dec_ops_s g_dec_s[] =
```

Comment:
can we use reference to `g_dec_pcm` and `g_dec_mp3` instead of copy?

Reply:
hello pkarashchenko, 
there is no copy, use the reference of g_dec_pcm or g_dec_mp3 in function `nxplayer_playinternal`.

Reply:
let fill the array in place

Reply:
> hello pkarashchenko, 
> there is no copy, use the reference of g_dec_pcm or g_dec_mp3 in function `nxplayer_playinternal`.
> 
> 

But `g_dec_s` array contain the copy of those structs, isn't it?